server/api/router.go

```diff
@@ -141,6 +141,8 @@ func createRouter(prefix string, svr *server.Server) *mux.Router {
 	router.Handle("/api/v1/health", newHealthHandler(svr, rd)).Methods("GET")
 	router.Handle("/api/v1/diagnose", newDiagnoseHandler(svr, rd)).Methods("GET")
 
+	router.HandleFunc("/api/v1/recovery/tso", newRecoveryHandler(handler, rd).ResetTS).Methods("POST")
```

Comment:
`recovery` is just a single feature, it should not occupy the api's name. What if there is a new feature that needs to reset the TSO too?